travis/analyze.sh

```diff
@@ -13,11 +13,11 @@ RESULTS=`dartanalyzer                                                          \
   out/host_debug_unopt/gen/sky/bindings/dart_ui/ui.dart                        \
   2>&1                                                                         \
   | grep -v "\[error\] Native functions can only be declared in the SDK and code that is loaded through native extensions" \
-  | grep -Ev "\[(hint|error)\] The function '.+' is not used"                  \
+  | grep -Ev "\[(hint|error)\] The function '.+' (is not|isn't) used"                  \
   | grep -Ev "\[(warning|error)\] Undefined name 'main'"                       \
   | grep -Ev "\[(warning|error)\] Undefined name 'VMLibraryHooks"              \
   | grep -v "\[error\] The library ''dart:_internal'' is internal"             \
-  | grep -Ev "Unused import .+ui\.dart"                                        \
+  | grep -Ev "Unused import.+ui\.dart"                                        \
```

Comment:
The new format has a period after "Unused import"

See e.g. https://github.com/dart-lang/sdk/commit/99ce88f8bc2e189ada41bc36c54a1830f32b9538